README.md

```diff
@@ -1 +1,27 @@
-Implements optimal ate pairings over the bn\_128 curve.
+py_ecc
+==========================
+Elliptic curve crypto in python including secp256k1 and alt_bn128
+
+[![GitHub](https://img.shields.io/github/license/mashape/apistatus.svg)](https://github.com/ethereum/py_ecc) ![Travis (.org) branch](https://img.shields.io/travis/ethereum/py_ecc/master.svg)[![PyPI version](https://badge.fury.io/py/py_ecc.svg)](https://badge.fury.io/py/py_ecc)
+
+
+## Quickstart
+```sh
+pip install py_ecc
+```
+
+## Developer Setup
+
+If you would like to hack on py_ecc, please check out the [Ethereum Development Tactical Manual](https://github.com/ethereum/ethereum-dev-tactical-manual) for information on how we do:
```

Comment:
Should probably use https://github.com/ethereum/snake-charmers-tactical-manual instead, even though the other url works.

Reply:
even I was going to changes, haha. @pipermerriam thanks for confirming :+1: